docs/iris/src/userguide/regridding_plots/regridding_plot.py

```diff
@@ -22,6 +22,4 @@
 ax.coastlines(resolution="50m")
 ax.gridlines()
 
-plt.tight_layout()
-
```

Comment:
@tkknight The `plt.tight_layout()` was causing a traceback within `matplotlib`, resulting in the plot not being rendered in the documentation i.e., see [here](https://scitools-iris.readthedocs.io/en/latest/userguide/interpolation_and_regridding.html#regridding).